### PR TITLE
Storybook: Modal View with Title and closing button X #19

### PR DESCRIPTION
**Pull request by 1620prabn**
# Description

Modal view with title and closing X button created

Fixes # (19)

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [ x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/Modal/Modal.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+* {
```
Comment (tpetrina) [inline src/client/components/Modal/Modal.css:1]: Global styles do not belong in a component CSS. Use a separate PR for that and adjust it in the global CSS.

File: src/client/components/Modal/Modal.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  justify-content: right;
+}
+
+.modal-content {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
+    Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  text-align: right;
+  font-size: 24px;
+}
+h4 {
```
Comment (tpetrina) [inline src/client/components/Modal/Modal.css:41]: This CSS selector isn't specific enough, please provide a better selector.

**Review by tpetrina — CHANGES_REQUESTED** [review]

File: src/client/components/Modal/Modal.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import PropTypes from 'prop-types';
+
+export default function Modal({ title }) {
+  return (
+    <div className="modal-wrapper">
+      <div className="modal-header">
+        <span className="close-modal-btn">x</span>
+      </div>
+      <h4>{title} </h4>
+    </div>
```
Comment (tpetrina) [inline src/client/components/Modal/Modal.js:12]: Consider also passing `children` property in this component.

**Comment by nuha90** [comment]
you made all the changes ,good work

**Review by nuha90 — DISMISSED** [review]

File: src/client/components/Modal/Modal.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        <span className="close-modal-btn">x</span>
+      </div>
+      <h4 className="modal-title">{props.title} </h4>
+      {props.children}
+    </div>
+  );
+}
+Modal.propTypes = {
+  title: PropTypes.string.isRequired,
+  children: PropTypes.string.isRequired,
```
Comment (tpetrina) [inline src/client/components/Modal/Modal.js:18]: This should be `node`, not `string`.

File: src/client/components/Modal/Modal.stories.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import Modal from './Modal';
+
+export default { title: 'Modal' };
+
+export const FeedbackModal = () => <Modal title="Feedback Form" />;
```
Comment (tpetrina) [inline src/client/components/Modal/Modal.stories.js:6]: If you specified `children` as required, why didn't you use it here?
Add it as an example.

**Review by tpetrina — CHANGES_REQUESTED** [review]
Great work, just a small step towards the end.

**Review by tpetrina — APPROVED** [review]
Good work!

**Review by Elia-Git — APPROVED** [review]
Good job!

**Pull request merged**